### PR TITLE
CLI options for kubeconfig file and context

### PR DESCRIPTION
**Pull request by wazir-ahmed**
Signed-off-by: Wazir Ahmed <wazir@accuknox.com>

### CODE REVIEW

File: k8s/client.go [hunk 1]
```diff
@@ -27,16 +27,16 @@ type Client struct {
 	Config          *rest.Config
 }
 
+var KubeConfig string
+var ContextName string
```
Comment (daemon1024) [inline k8s/client.go:31]:
```suggestion
// K8s Config and Context data
var (
    KubeConfig string
    ContextName string
)
```

Pleasing the Linter :)

Comment (wazir-ahmed) [reply k8s/client.go]: Done :+1:

Comment (daemon1024) [reply k8s/client.go]: Ahh it needs the comments, else the linter will keep warning 😅 Can check the files tab.

Comment (wazir-ahmed) [reply k8s/client.go]: Added the comments.

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**